src/AppBundle/Entity/Report/Traits/StatusTrait.php

```diff
@@ -46,28 +60,43 @@ public function setStatusCached(array $status)
         $this->statusCached = json_encode($status);
     }
 
+    /**
+     * @return string
+     */
+    public function getReportStatusCached()
+    {
+        return $this->reportStatusCached;
+    }
+
```

Comment:
simple getter of the reportStatusCached, discussed above. It's not even used as hydrates  as array, but I'll keep it just in case I forgot something and avoid re-running tests, flagged for refactor at next iteration